solr/core/src/java/org/apache/solr/cloud/api/collections/RestoreCmd.java

```diff
@@ -67,6 +49,26 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.Closeable;
```

Comment:
Why move the `java` imports to after `org.apache`? Usually the fundamental imports are first, then the rest.

Reply:
Definitely no intentional reason - I suspect I just have some variation in my IDE's code-editor settings.

I've bootstrapped my IntelliJ with what comes out of `ant idea` though, so not sure how that could've arisen?  But I'll take a look to make sure I'm not out of line.

Reply:
I later realized my IDEA was doing the same thing and I've checked in files too with the same issue.
I since changed manually the config (settings - editor - java and then at the bottom you can define the ordering and spacing).